goas/cl/cl/src/nullifier.rs

```diff
@@ -25,13 +25,13 @@ pub struct NullifierSecret(pub [u8; 16]);
 // can be provided to anyone wishing to transfer
 // you a note
 #[derive(Debug, Clone, Copy, PartialEq, Eq, Serialize, Deserialize)]
-pub struct NullifierCommitment([u8; 32]);
+pub struct NullifierCommitment(pub [u8; 32]);
```

Comment:
maybe `From` implementation?

Reply:
Turns out I don't need this actually, reverted.